source/Junkership.js

```diff
@@ -10,7 +10,7 @@ import Sound from "./Sound.js"
 import {PowerUp, PeaShoota, TriShoota, FiveShoota, RapidFire, RapidSprayShot,
     SprayShot, SuperSprayShot, CrazySprayShot, VertSprayShot,
     VertShoota, BFG, Laser, PiercingLaser, SuperLaser, Mine,
-    SuperMine, PaintShot} from "./PowerUp.js"
+    SuperMine, PaintShot, Sword, Shield} from "./PowerUp.js"
```

Comment:
Looks like when you did the merge conflicts you took the wrong line. Only the weapon that the Junkership starts with is needed. This was to remove dependencies. Thus, with you starting with only Sword right now, this line should read `import { Sword } from "./PowerUp.js"`, and just change it back to PeaShoota in the final commit.